src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -4876,6 +4876,149 @@
     public static final int SWITCH_RULE =
             GeneratedJavaTokenTypes.SWITCH_RULE;
 
+    /**
+     * The {@code non-sealed} keyword.  This element appears
+     * as part of a class or interface declaration.
+     *
+     * <p>For example:</p>
+     * <pre>
+     * non-sealed class Square extends Rectangle { }
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * CLASS_DEF -&gt; CLASS_DEF
+     * |--MODIFIERS -&gt; MODIFIERS
+     * |   `--LITERAL_NON_SEALED -&gt; non-sealed
+     * |--LITERAL_CLASS -&gt; class
+     * |--IDENT -&gt; Square
+     * |--EXTENDS_CLAUSE -&gt; extends
+     * |   `--IDENT -&gt; Rectangle
+     * `--OBJBLOCK -&gt; OBJBLOCK
+     *     |--LCURLY -&gt; {
+     *     `--RCURLY -&gt; }
+     * </pre>
+     *
+     * @see <a href="https://docs.oracle.com/en/java/javase/15/docs/specs/sealed-classes-jls.html">
+     * Java Language Specification, &sect;8.1.7</a>
+     * @see #MODIFIERS
+     *
+     * @since 8.43
```

Comment:
next release is 8.42

Reply:
I wasn't optimistic submitting a PR for a grammar change during pre-GSOC :) I will fix this since we are close to merging now.

Reply:
Fixed.